### PR TITLE
Add vuid 06540, Validate depthAttachmentFormat has depth aspect

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -8004,6 +8004,13 @@ bool CoreChecks::PreCallValidateBeginCommandBuffer(VkCommandBuffer commandBuffer
                             "vkBeginCommandBuffer(): VkCommandBufferInheritanceRenderingInfo->depthAttachmentFormat (%s) must be a format with potential format features that include VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT",
                             string_VkFormat(depth_format));
                     }
+                    if (!FormatHasDepth(depth_format)) {
+                        skip |= LogError(
+                            commandBuffer, "VUID-VkCommandBufferInheritanceRenderingInfo-depthAttachmentFormat-06540",
```
Comment (sjfricke) [inline layers/core_validation.cpp:8009]: nit: update the PR title to say VUID 06540

File: layers/core_validation.cpp [hunk 2]
```diff
@@ -8004,6 +8004,13 @@ bool CoreChecks::PreCallValidateBeginCommandBuffer(VkCommandBuffer commandBuffer
                             "vkBeginCommandBuffer(): VkCommandBufferInheritanceRenderingInfo->depthAttachmentFormat (%s) must be a format with potential format features that include VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT",
                             string_VkFormat(depth_format));
                     }
+                    if (!FormatHasDepth(depth_format)) {
```
Comment (sjfricke) [inline layers/core_validation.cpp:8007]: nit: I think this PR is find as is, but the `valid_depth_stencil_format` variable could probably just be inlined into the above `if` statement now

File: layers/core_validation.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply layers/core_validation.cpp:8009]: Not sure how that happened, good catch

File: layers/core_validation.cpp (hunk 2 above)
Comment (ziga-lunarg) [reply layers/core_validation.cpp:8007]: It's still used in 2 places and not related to this PR

Comment (sjfricke) [reply layers/core_validation.cpp:8007]: forgive me, was lazy to pull up my text editor and used GitHub webivew :laughing: 

All good then too me!

**Review by sjfricke — APPROVED** [review]

**Pull request merged**